app/controllers/root_controller.rb

```diff
@@ -50,15 +50,17 @@ def random_page
     # new content.
     expires_in 5.seconds
 
-    base_url = "https://www.gov.uk/api/artefacts.json"
-    total_pages = JSON.parse(RestClient.get(base_url).body)['pages']
+    base_url = "https://www.gov.uk/api/search.json"
+    total_pages = JSON.parse(RestClient.get("#{base_url}?count=0"))['total']
```

Comment:
You might want to change the variable names as this isn't the total pages, it's the total documents in Rummager.